auth0/resource_auth0_prompt_custom_text.go

```diff
@@ -0,0 +1,162 @@
+	}
+	availableLanguages = []string{
+		"ar", "bg", "bs", "cs", "da", "de", "el", "en", "es", "et", "fi", "fr", "fr-CA", "fr-FR", "he", "hi", "hr",
+		"hu", "id", "is", "it", "ja", "ko", "lt", "lv", "nb", "nl", "pl", "pt", "pt-BR", "pt-PT", "ro", "ru", "sk",
+		"sl", "sr", "sv", "th", "tr", "uk", "vi", "zh-CN", "zh-TW",
+	}
+	errEmptyPromptCustomTextID         = fmt.Errorf("ID cannot be empty")
+	errInvalidPromptCustomTextIDFormat = fmt.Errorf("ID must be formated as prompt:language")
+)
+
+func newPromptCustomText() *schema.Resource {
+	return &schema.Resource{
+		Create: createPromptCustomText,
+		Read:   readPromptCustomText,
+		Update: updatePromptCustomText,
+		Delete: deletePromptCustomText,
+		Importer: &schema.ResourceImporter{
+			State: importPromptCustomText,
+		},
+		Schema: map[string]*schema.Schema{
+			"prompt": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringInSlice(availablePrompts, false),
+			},
+			"language": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringInSlice(availableLanguages, false),
+			},
+			"body": {
+				Type:             schema.TypeString,
+				Required:         true,
+				ValidateFunc:     validation.StringIsJSON,
+				DiffSuppressFunc: structure.SuppressJsonDiff,
+			},
+		},
+	}
+}
+
+func importPromptCustomText(d *schema.ResourceData, _ interface{}) ([]*schema.ResourceData, error) {
+	prompt, language, err := getPromptAndLanguage(d)
+
+	d.SetId(d.Id())
+	d.Set("prompt", prompt)
+	d.Set("language", language)
+
+	return []*schema.ResourceData{d}, err
+}
+
+func createPromptCustomText(d *schema.ResourceData, m interface{}) error {
+	d.SetId(d.Get("prompt").(string) + ":" + d.Get("language").(string))
+	return updatePromptCustomText(d, m)
+}
+
+func readPromptCustomText(d *schema.ResourceData, m interface{}) error {
+	api := m.(*management.Management)
+	customText, err := api.Prompt.CustomText(d.Get("prompt").(string), d.Get("language").(string))
+	if err != nil {
+		if mErr, ok := err.(management.Error); ok {
+			if mErr.Status() == http.StatusNotFound {
+				d.SetId("")
+				return nil
+			}
+		}
+		return err
+	}
+
+	body, err := marshalCustomTextBody(customText)
+	if err != nil {
+		return err
+	}
+
+	d.Set("body", body)
+	return nil
+}
+
+func updatePromptCustomText(d *schema.ResourceData, m interface{}) error {
+	api := m.(*management.Management)
+	prompt, language, err := getPromptAndLanguage(d)
+	if err != nil {
+		return err
+	}
+
+	if d.Get("body").(string) != "" {
+		var body map[string]interface{}
+		if err := json.Unmarshal([]byte(d.Get("body").(string)), &body); err != nil {
+			return err
+		}
+
+		err := api.Prompt.SetCustomText(prompt, language, body)
+		if err != nil {
+			return err
+		}
+	}
+
+	return readPromptCustomText(d, m)
+}
+
+func deletePromptCustomText(d *schema.ResourceData, m interface{}) error {
+	d.Set("body", "{}")
+
+	updatePromptCustomText(d, m)
```

Comment:
Let's please make sure we don't ignore this error:) 
```suggestion
	if err := updatePromptCustomText(d, m); err != nil {
		return err
	}
```